backend/stackexchange/stack_api.py

```diff
@@ -0,0 +1,45 @@
+from datetime import datetime
+from stackapi import StackAPI
+from stackapi import StackAPIError
+import json
+
+
+class UserData:
```

Comment:
should be a data class, extract work with stack exchange API to repository layer

Reply:
done, need to review